__tests__/MapScreen-test.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { render } from "@testing-library/react-native";
+import MapScreen from "@/app/screens/map/map_screen";
+import * as Location from "expo-location";
+
```

Comment:
There should be tests for edge cases, for example : empty collection, or challenges that don't have all the fields that are valid.

Reply:
I'll add more tests.